must-gather/collection-scripts/gather

```diff
@@ -43,9 +43,10 @@ commands+=("get sc")
 commands+=("get infrastructures.config -oyaml")
 
 # Run the Collection of Resources to list
-for command in "${commands[@]}"; do
-     echo "collecting oc command ${command}" | tee -a ${BASE_COLLECTION_PATH}/gather-debug.log
-     timeout 120 oc ${command}  >> ${BASE_COLLECTION_PATH}/oc_output/"${command}"
+for ((i = 0; i < ${#commands[@]}; i++)); do
```

Comment:
explain this changes

Reply:
seems like we are not logging the echo to gather-debug.log

Reply:
@ashishkumsingh I agree with @crombus, please add a description to your commit message (not the PR) explaining why you are making this change and why you had to change this `for` loop.

Reply:
# Run the Collection of Resources to list
for command in "${commands[@]}"; do
    echo "collecting oc command ${command}" | tee -a ${BASE_COLLECTION_PATH}/gather-debug.log
    COMMAND_OUTPUT_FILE=${BASE_COLLECTION_PATH}/oc_output/${command// /_}
    timeout 120 oc ${command} >> "${COMMAND_OUTPUT_FILE}"
done

I suggest to do these changes. It works fine and not much of code change is required.

Reply:
@jarrpa noted. I have included them in the commit message now.

@crombus due to a typo (missing brackets) in my local repo, I was not able to proceed with the existing `loop`. However, the snippet you posted works fine. I have included that.